README.md

```diff
@@ -30,13 +30,55 @@ The concrete class that implements `IStatsDPublisher` is `StatsDPublisher`. The
 
 An example of a very simple StatsD publisher configuration, using the default values for most things.
 
+#### .NET Core
+
 ```csharp
+// Service registration
+services.AddStatsD("metrics_server.mycompany.com");
+services.AddSingleton<MyService>();
+
+// Consumption
+public class MyService
+{
+    public MyService(IStatsDPublisher statsPublisher)
+    {
+        // Use the publisher
+    }
+}
+
+```
+
+#### .NET 4.5.1
```

Comment:
You can still do this .Net core. You just aren't taking advantage of the service registration bits.

Reply:
Suggested alternate heading?

Reply:
Hmmm, maybe

#### Simplest example
No service registration or Ioc. Works in .NET 4.5.1 and .NET core.